test/fuzz/transformation_add_parameter_test.cpp

```diff
@@ -188,18 +205,19 @@ TEST(TransformationAddParameterTest, NonPointerBasicTest) {
          %41 = OpTypeStruct %11 %16
          %42 = OpConstantComposite %41 %8 %32
          %44 = OpTypeFunction %2 %41 %7
-          %6 = OpTypeFunction %7 %7 %11
+          %6 = OpTypeFunction %7 %7 %11 %52
          %65 = OpTypeFunction %2 %31
```

Comment:
In the test output, instructions %6 and %65 are in the opposite order.

Can you run your test in the debugger, and step through each call to `fuzzerutil::UpdateFunctionType`? This should give you insight into when existing function types are changed, or new function types added. In your case, I think the existing function type %65, is being changed (your transformation proposed id %69 for a new function type, if one were required, but it looks like no new type was required in this case).

Reply:
Ok, will do. Thank you for the comments!

Reply:
Done!